lib/src/components/NxList/__tests__/NxListButtonItem.test.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxListButtonItemProps } from '../types';
+import NxList from '../NxList';
+import { getMountedComponent, getShallowComponent } from '../../../__testutils__/enzymeUtils';
+
+describe('NxListButtonItem', function() {
+
+  const minimalProps: NxListButtonItemProps = {};
+  const getShallow = getShallowComponent(NxList.ButtonItem, minimalProps);
+  const getMounted = getMountedComponent(NxList.ButtonItem, minimalProps);
+
+  it('renders the classNames given to it', function() {
```

Comment:
We need a test for the general structure of the component - a button within an li.  This test kind of covers that, but not super well, and the test name doesn't hint at it

Reply:
> We need a test for the general structure of the component - a button within an li. This test kind of covers that, but not super well, and the test name doesn't hint at it

Added tests to check for the correctness of the general structure of both button and list components.
f86d157bba1c8df0e827430661e3b603ca09dd8b